src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -0,0 +1,3118 @@
+                <xbrli:entity>
+                  <xbrli:identifier scheme="http://www.companieshouse.gov.uk/">{{$companyNumber}}</xbrli:identifier>
+                  <xbrli:segment>
+                    <xbrldi:explicitMember dimension="core:RelatedPartiesDimension">core:Associate11</xbrldi:explicitMember>
+                  </xbrli:segment>
+                </xbrli:entity>
+                <xbrli:period>
+                  <xbrli:startDate>{{$current_period_start_on}}</xbrli:startDate>
+                  <xbrli:endDate>{{$current_period_end_on}}</xbrli:endDate>
+                </xbrli:period>
+              </xbrli:context>
+              {{if $previous_period_start_on}}
+                <xbrli:context id="RelatedParty11_PY_END">
+                  <xbrli:entity>
+                    <xbrli:identifier scheme="http://www.companieshouse.gov.uk/">{{$companyNumber}}</xbrli:identifier>
+                    <xbrli:segment>
+                      <xbrldi:explicitMember dimension="core:RelatedPartiesDimension">core:Associate11</xbrldi:explicitMember>
+                    </xbrli:segment>
+                  </xbrli:entity>
+                  <xbrli:period>
+                    <xbrli:instant>{{$previous_period_end_on}}</xbrli:instant>
+                  </xbrli:period>
+                </xbrli:context>
+              {{end}}
+              <xbrli:context id="RelatedParty11_CY_END">
+                <xbrli:entity>
+                  <xbrli:identifier scheme="http://www.companieshouse.gov.uk/">{{$companyNumber}}</xbrli:identifier>
+                  <xbrli:segment>
+                    <xbrldi:explicitMember dimension="core:RelatedPartiesDimension">core:Associate11</xbrldi:explicitMember>
+                  </xbrli:segment>
+                </xbrli:entity>
+                <xbrli:period>
+                  <xbrli:instant>{{$current_period_end_on}}</xbrli:instant>
+                </xbrli:period>
+              </xbrli:context>
+               <xbrli:context id="RelatedParty12_CY">
+                <xbrli:entity>
+                  <xbrli:identifier scheme="http://www.companieshouse.gov.uk/">{{$companyNumber}}</xbrli:identifier>
+                  <xbrli:segment>
+                    <xbrldi:explicitMember dimension="core:RelatedPartiesDimension">core:Associate12</xbrldi:explicitMember>
+                  </xbrli:segment>
+                </xbrli:entity>
+                <xbrli:period>
+                  <xbrli:startDate>{{$current_period_start_on}}</xbrli:startDate>
+                  <xbrli:endDate>{{$current_period_end_on}}</xbrli:endDate>
+                </xbrli:period>
+              </xbrli:context>
+              {{if $previous_period_start_on}}
+                <xbrli:context id="RelatedParty12_PY_END">
+                  <xbrli:entity>
+                    <xbrli:identifier scheme="http://www.companieshouse.gov.uk/">{{$companyNumber}}</xbrli:identifier>
+                    <xbrli:segment>
+      
```

Comment:
Loans to director is not part of the MVP. These iXBRL elements will end up in the generated document.